tests/test-helper.js

```diff
@@ -1,8 +1,29 @@
-import Application from '../app';
-import config from '../config/environment';
-import { setApplication } from '@ember/test-helpers';
-import { start } from 'ember-qunit';
+import QUnit from 'qunit';
+import AbstractTestLoader from 'ember-cli-test-loader/test-support/index';
```

Comment:
im curious why we are using the base testloader here instead of using ember-qunit